FPM.ftd

```diff
@@ -7,11 +7,14 @@ zip: github.com/fifthtry/df/archive/refs/heads/main.zip
 
 -- fpm.dependency: fifthtry.github.io/fifthtry-brand-switcher as bs
 
+-- fpm.dependency: fifthtry.github.io/forest-cs as frst
+
 
 
 -- fpm.auto-import: ds
 -- fpm.auto-import: df.fifthtry.com/FPM/config
 -- fpm.auto-import: df.fifthtry.com/assets
+-- fpm.auto-import: frst
```

Comment:
No need to give it an alias. Use full name. Let's use alias only if something is frequently used. Also no need to put it in auto import. Add an explicit import in config.ftd.